config.example/group_vars/all.yml

```diff
@@ -66,3 +66,14 @@
 
 #software_remove_packages:
 #  - popularity-contest
+
+# users configuration
+# create user `nvidia` with password `deepops` on all nodes
+# run with user-password.yml playbook
```

Comment:
Should be `playbooks/users.yml` vs `playbooks/user-password.yml`, which has a hard-coded user var

Reply:
Done